### PR TITLE
dx 469 new screenshots

### PR DESCRIPTION
**Pull request by arebra**
Added new screenshots for everything with a standardized width.

### CODE REVIEW

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6126346&bid=20040225)

File: _includes/merchant-authenticated-consumer.md [hunk 1]
```diff
@@ -57,12 +57,13 @@ not require the last 4/5 digits in the payer's social security number (SSN), for
...
-without Merchant Authenticated Consumer. To the left, the SSN is removed
-altogether because the payer is authenticated by the Merchant, and to the right
-the last 4/5 digits of the SSN required as usual.
-
-![One-Click Payments without SSN][mac-no-ssn]{:width="436" height="393"}
-![Payments with SSN][mac-with-ssn]{:width="436" height="393"}
+without Merchant Authenticated Consumer. In the first image, the SSN is removed
+altogether because the payer is authenticated by the Merchant. In the bottom
+image, the last 4/5 digits of the SSN are required as usual.
+{:.text-center}
```
Comment (NullableInt) [inline _includes/merchant-authenticated-consumer.md:63]:
```suggestion
image, the last 4/5 digits of the SSN are required as usual.

{:.text-center}
```
A bit of air between text and a image makes it more readable.

File: _includes/payment-link.md [hunk 2]
```diff
@@ -84,8 +85,8 @@ set to `Purchase` are listed below.
 
 When clicking the payment link, the consumer will be directed to a payment
 page, similar to the examples below, where payment information can be entered.
-
-![screenshot of the redirect card payment page][card-payment]{:height="500px" width="425px"}
+{:.text-center}
+![screenshot of the redirect card payment page][card-payment]{:height="620px" width="475px"}
```
Comment (NullableInt) [inline _includes/payment-link.md:89]:
```suggestion

{:.text-center}
![screenshot of the redirect card payment page][card-payment]{:height="620px" width="475px"}
```
Just suggesting a bit of air between text and image

**Review by NullableInt — COMMENTED** [review]
Just a few comments for your consideration. 😺

**Comment by NullableInt** [comment]
@asbjornu if `{:.text-center}` is needed for all images, should it be the new default?

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6126346&bid=20040225)

**Review by asbjornu — APPROVED** [review]
👏

**Review by NullableInt — APPROVED** [review]

**Pull request merged**